e2e/protractor.gh-chrome.conf.js

```diff
@@ -0,0 +1,14 @@
+// Protractor configuration file, see link for more information
```

Comment:
Let's add a brief header comment describing the file's purpose:
e.g. `Used to run e2e tests with Chrome (in headless mode)`

Reply:
I'll also add it to the `gh-firefox` config 👍